x-pack/plugins/infra/public/components/alerting/metrics/metric_threshold_alert_type.ts

```diff
@@ -23,10 +23,10 @@ export function getAlertType(): AlertTypeModel {
     defaultActionMessage: i18n.translate(
       'xpack.infra.metrics.alerting.threshold.defaultActionMessage',
       {
-        defaultMessage: `\\{\\{alertName\\}\\} - \\{\\{context.group\\}\\}
+        defaultMessage: `\\{\\{alertName\\}\\} - \\{\\{context.group\\}\\} is in a state of \\{\\{context.alertState\\}\\}
 
-\\{\\{context.metricOf.condition0\\}\\} has crossed a threshold of \\{\\{context.thresholdOf.condition0\\}\\}
-Current value is \\{\\{context.valueOf.condition0\\}\\}
+Reason:
```

Comment:
Any reason to have a line break after `Reason:`? It reads a little strange in a server log because it adds an extra semicolon. Also, feels like we might not even need to have `Reason:` at all. Just the message might be enough.

Example server log: `No Data alert for Aerospike - * is in a state of NO DATA;;Reason:;aerospike.namespace.memory.free.pct has reported no data over the past 1m;;NO DATA`

Reply:
We wanted something besides just `{{context.reason}}` in the default message because right now we have no other way to tutorialize what the context variables mean. If a user comes across a text field that says:

```
{{alertName}} - {{context.group}} is in a state of {{context.alertState}}

{{context.reason}}
```

it's not very clear what that means if you're not already familiar.

We added the line break under the assumption that multiple conditions would be separated by more line breaks, and it'd look better if the first condition was also on a new line.

It definitely looks weird in server logs, I agree. I'm not sure why we don't just allow server logs to format with line breaks.

IMO I think this is the best we can do the way the actions UI is right now, and we should talk about a broader redesign.

Reply:
Ok, that makes sense.